libs/data-display/src/useSortableDataTable.tsx

```diff
@@ -0,0 +1,21 @@
+import _ from "lodash";
+
+import { useDataTable } from "./index";
+
+export default function useSortableDataTable<T, U extends keyof T>(initialData: T[], columnMapping: Record<U, string>) {
+  const [dataTableState, dataTableHook] = useDataTable();
+
+  const generateSortedData = () => {
+    const sortInfo = dataTableState.sortBy[0];
```

Comment:
This is only sorted by first sort info, others are ignores. Are we even providing functionality of multi sort?

Reply:
The current sorting mechanism is primarily designed to handle single-column sorting. While the initial render correctly considers the first sort instruction, subsequent sorting operations lack effective handling for multiple columns. To address this, we're enhancing the sorting algorithm in ticket #182 . The proposed changes ensure a more robust multi-column sorting experience, allowing users to click on different columns without affecting the sorting state of others not actively selected in the current interaction.